openapi/geonames/Module.md

```diff
@@ -0,0 +1,36 @@
+## Overview
+
+This is a generated connector for [Geonames API](https://www.geonames.org/export/JSON-webservices.html) OpenAPI specification.
+The Geonames API allows you to retrieve information about geo locations. . 
```

Comment:
```suggestion
The Geonames API allows you to retrieve information about geo locations. 
```

Reply:
Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/45c0ec66caf0d3ead895a90f7f2ff2ef6c431654